iosApp/iosApp/Pages/Search/Results/StopResultsView.swift

```diff
@@ -40,9 +40,11 @@ struct StopResultsView: View {
                     .foregroundStyle(Color.text)
                     .multilineTextAlignment(.leading)
                     .frame(maxWidth: .infinity, alignment: .leading)
-                HStack(spacing: 4) {
-                    ForEach(stop.routes, id: \.self) { route in
-                        RoutePill(stopResultRoute: route, type: .flex)
+                ScrollView(.horizontal, showsIndicators: false) {
```

Comment:
I think this will still scroll and capture drags even if the content is to short to actually be off the screen, can we add a `.scrollBounceBehavior(.basedOnSize, axes: [.horizontal])` like in [TripDetailsStopView](https://github.com/mbta/mobile_app/blob/main/iosApp/iosApp/Pages/TripDetails/TripDetailsStopView.swift#L56)?